generator/src/_Azure.Generator.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <PackageId>_AzurePlugin</PackageId>
+    <Version>1.0.0-beta.1</Version>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+  </PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.Generator.CSharp.ClientModel" version ="1.0.0-alpha.20240820.5"/>
+	</ItemGroup>
+
+	<!-- Copy output to package dist path for local execution and  -->
+	<Target Name="CopyForNpmPackage" AfterTargets="Build">
+		<Message Text="Copying output to dist path" Importance="high" />
+		<ItemGroup>
+			<SourceDir Include="$(OutputPath)**\*.*" />
+		</ItemGroup>
+		<Copy SourceFiles="@(SourceDir)" DestinationFolder="$(MSBuildThisFileDirectory)..\..\dist\generator\%(RecursiveDir)" />
+	</Target>
+
+	<PropertyGroup>
```

Comment:
disable the central package management to be able to use non-stable packages as dependency

Reply:
`VersionOverride` fixes it for now